Data/InstructionsForAddingUser.txt

```diff
@@ -0,0 +1,5 @@
+﻿1) Create User controller - post method, takes user as request, returns response class
+2) Create user class - contains all fields
```

Comment:
delete this file

Reply:
Oh yeah I forgot to remove it
I'll do that